subsys/usb/device/class/audio/audio.c

```diff
@@ -354,6 +354,19 @@ static void audio_cb_usb_status(struct usb_cfg_data *cfg,
 	}
 }
 
+static int16_t round_vol_to_closest(int16_t target_vol)
+{
+	int rounded_vol  = 0;
+	int16_t step = (int16_t)CONFIG_USB_DEVICE_AUDIO_RES_VOLUME;
```

Comment:
Why is there (int16_t) cast?

Reply:
This function has been removed since I switched to use ROUND_UP from sys/utils.h